examples/models/llava/runner/llava_text_decoder_runner.h

```diff
@@ -17,7 +17,8 @@ namespace example {
 class ET_EXPERIMENTAL LlavaTextDecoderRunner
     : public executorch::extension::llm::TextDecoderRunner {
  public:
-  explicit LlavaTextDecoderRunner(executorch::extension::Module* module)
+  explicit LlavaTextDecoderRunner(
+      std::shared_ptr<executorch::extension::Module> module)
       : TextDecoderRunner(module, true) {}
```

Comment:
need to `std::move(module)`

Reply:
Do we do std::move for shared_ptr as well? I always thought only unique_ptr uses it.